client/rpcserver/handlers.go

```diff
@@ -452,4 +461,71 @@ Registration is complete after the fee transaction has been confirmed.`,
 		returns: `Returns:
     string: The message "` + fmt.Sprintf(feePaidStr, "[fee]") + `"`,
 	},
+	exchangesRoute: {
+		pwArgsShort: ``,
+		argsShort:   ``,
+		cmdSummary:  `List assets supported by all registered DEX servers their and active trades.`,
+		pwArgsLong:  ``,
+		argsLong:    ``,
+		returns: `Returns:
+    map: The exchanges result.
+    {
+      "[DEX url] (map): {
+        "url" (string): The DEX URL.
+        "markets" (map): {
+          "[assetID-assetID]" (map): {
+            "name" (string): The market name. e.g. "dcr_btc".
+            "baseid" (int): The base assetID
+            "basesymbol" (string): The base assetID symbol.
+            "quoteid" (int): The quote assetID.
+            "quotesymbol" (string): The quote assetID symbol,
+            "epochlen" (int): Duration of a trade window in milliseconds.
+            "startepoch" (int): Unix time of start of the last epoch in seconds
+	      since 00:00:00 Jan 1 1970.
+            "buybuffer" (float): The minimum order size for a market buy order.
+            "orders" (map): {
+              "dex" (string): The DEX URL.
+              "market" (string): The market name. e.g. "dcr_btc".
+              "type" (int): 0 for market or 1 for limit.
+              "id" (string): A unique trade ID.
+              "stamp" (int): The unix trade timestamp. Seconds since 00:00:00
+	        Jan 1 1970.
+              "qty" (bool): Number of units offered in the trade.
+              "sell" (bool): Whether this is a sell order.
+              "filled" (int): How much of the order has been filled.
+              "matches" (array): [
+	        {
+                  "matchID" (string): A unique match ID.
+                  "step" (string): ???
```

Comment:
Match status. Maybe should change the json tag to `"status"`. I don't think the GUI is using the field right now, so should be okay.